### PR TITLE
docs(Readme): Add VS Code integration

### PR DESCRIPTION
**Pull request by zachverkstedt**

### CODE REVIEW

**Comment by marek-saji** [comment]
@zachverkstedt Please request review from somebody that uses VSCode. :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -47,3 +47,28 @@ ESLint and Prettier config
 We recommend to use [CoC][vim-coc] with `coc-eslint` extension.
 
 [vim-coc]: https://github.com/neoclide/coc.nvim
+
+## VS Code Integration
+
+To get this working with VS Code, you will first need to install the Prettier - Code formatter extension: `https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode`.
```
Comment (marek-saji) [inline README.md]:
```suggestion
To get this working with VS Code, you will first need to install the [Prettier - Code formatter extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
```

File: README.md [hunk 2]
```diff
@@ -47,3 +47,28 @@ ESLint and Prettier config
...
+  "[javascriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  }
+```
```
Comment (sven-verkstedt) [inline README.md]: I would suggest not to put those settings into global, but to recommend as per language. There are people working with other langs than JS/TS ;)

```suggestion
```json
  // probably best to use these only in workspace or
  // folder config, and not in global settings:
  "prettier.useEditorConfig": false,
  "eslint.format.enable": true,
  // Per language settings:
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true
  },
  "[javascriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true
  },
  "[typescript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true
  },
  "[typescriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true
  }
```
```

File: README.md [hunk 3]
```diff
@@ -47,3 +47,28 @@ ESLint and Prettier config
 We recommend to use [CoC][vim-coc] with `coc-eslint` extension.
 
 [vim-coc]: https://github.com/neoclide/coc.nvim
+
+## VS Code Integration
+
+To get this working with VS Code, you will first need to install the [Prettier - Code formatter extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+
+Next, add the following values to your `settings.json`:
```
Comment (sven-verkstedt) [inline README.md]:
```suggestion
Next, add the following values to global settings, or to to your per project folder settings (`./.vscode/settings.json`):
```

**Review by sven-verkstedt — APPROVED** [review]
Looks fine. My javascript config in VSCode also looks same (just have some additional `"editor.rulers": [80, 100, 120 ]` configured and that's it)
:+1:

**Pull request merged**